CHANGELOG.md

```diff
@@ -20,6 +20,12 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ### New features
 
+* The new config option `snapshot.auto-track` lets you automatically track only
+  the specified paths (all paths by default). Use the new `jj file track`
+  command to manually tracks path that were not automatically tracked. There is
+  no way to list untracked files yet.
```

Comment:
(Idle thinking) I think we'd want an option to `jj file list` that debugs why files are tracked (it could now be the `auto-track` pattern, explicit `track` command; we could also distinguish .gitignore patterns that start with `!`). Perhaps that'd make sense to combine that with a command that lists untracked files.